File: 7-kyu/16+18=214/javascript/solution.js
function add(num1, num2) {
  num1 = [...String(num1)].reverse()
  num2 = [...String(num2)].reverse()
  let limit = Math.max(num1.length, num2.length)
  let result = []
  for (var i = 0; i < limit; i++)
  {
    result.push((parseInt(num1[i]) || 0) + (parseInt(num2[i]) || 0))
  }
  return parseInt(result.reverse().join(""))
}

File: 7-kyu/16+18=214/ruby/solution.rb
def silly_add(a, b)
  num1 = a.to_s.split("").reverse()
  num2 = b.to_s.split("").reverse()
  result = []
  limit = [num1.length, num2.length].max
  limit.times { |i| 
    result.push((num1[i].to_i || 0) + (num2[i].to_i || 0))
  }
  return result.reverse().join("").to_i
end